packages/web-react/src/types/tag.ts

```diff
@@ -2,15 +2,15 @@ import { ElementType } from 'react';
 import {
   ChildrenProps,
   EmotionColorsDictionaryType,
-  SizesDictionaryType,
+  SizeExtendedDictionaryType,
   SpiritPolymorphicElementPropsWithRef,
   StyleProps,
   TransferProps,
 } from './shared';
 
 export type TagColor<C> = EmotionColorsDictionaryType | 'neutral' | C;
 
-export type TagSize<S> = SizesDictionaryType | S;
+export type TagSize<S> = SizeExtendedDictionaryType | S;
```

Comment:
This seems not correct to me. This changes API. Why we need this change?

Reply:
It actually extends the API. Web package already has it, Twig too and here it is documented that we also have it, but we actually didn't. 

It doesn't break or remove anything, it adds more options.